src/components/BarChart.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Bar } from '@nivo/bar';
+
+const BarChart = ({ data, xaxis, yaxis }) => (
+	<div
+		style={{
+			width: 'calc(100vw - 5rem)',
+			height: 630,
+			textAlign: 'center',
+			overflowX: 'scroll',
+			overflowY: 'hidden',
+			margin: '0 30px'
+		}}
```

Comment:
I hope you can move this and the other static styling into CSS as well!

I also recommend using `auto` instead of `scroll` for overflowX. On some browsers like Firefox, `scroll` will always show a scrollbar even if none is needed (it will just be a disabled scrollbar) while `auto` will only show a scrollbar when you can actually scroll.